examples/bundles/test.bundle.ts

```diff
@@ -0,0 +1,115 @@
+// Root process expects to be started with 'startContext = { maxRunTime: N }'
+
+interface POSISTest_BaseProcessMemory
+{
+	started?: number;
+	lastTick?: number;
+	supposedToBeDead?: boolean;
+
+	// set by start context
+	maxRunTime?: number;
+}
+
+class POSISTest_BaseProcess implements IPosisProcess
+{
+	public testName = "POSIS base:";
+	public static ImageName = "POSISTest/PosisBaseTestProcess";
+
+	constructor(private context: IPosisProcessContext)
+	{		
+	}
```

Comment:
`private context: IPosisProcessContext` creates private prop on this automatically via typescript magic